### PR TITLE
Beta 2023.05.29

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 14/14_01A_CIEL14A_4.txt [hunk 1]
```diff
@@ -505,7 +505,7 @@ That act alone is enough to make me forget that I'm even alive.
 [sha:27e720b7882eb2b1fb56b2bdc05f4b2938d519ac]{
 -- Page 130, Offset 37981.
 -- 「なんだ、ちゃんといるじゃない。
-"Oh, so you were here after all.
+"Oh, so you %{i}were%{/i} here after all.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01A_CIEL14A_4.txt:508]: Isn't how I'd emphasize it imo

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt [hunk 2]
```diff
@@ -339,7 +339,7 @@ There's no one around me.
 [sha:9e0fb5026add0807911423353d45b836fca9fb05]{
 -- Page 264, Offset 39132.
 -- 簡略化された街では喧噪もノイズもない。
-There's no hubbub or other noise coming from the simplified city, either.
+There's no hubbub or other noise coming from the simplified city either.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt:342]: Prefer the comma here

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt [hunk 3]
```diff
@@ -460,7 +460,7 @@ In other words, she's being pushed back!
 -- Page 301, Offset 39235.
 -- 鉄みたいに頑丈でつまらない女だと思ったけど。あんがい可愛いところがあるのね、シエル」
 I thought you were as boring and sturdy as a lump of iron,
- but you can be unexpectedly adorable at times, can't you, Ciel?"
+ but you can be unexpectedly adorable at times, aren't you, Ciel?"
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1G.txt:463]: Wrong, revert

File: script/Ciel/Day 14/14_01B_02_CIEL14B_4.txt [hunk 4]
```diff
@@ -184,7 +184,7 @@ This place isn't an office district anymore.
 [sha:cab28ba514ef827aee7bfa3786de4352d80745ee]{
 -- Page 259, Offset 40190.
 -- 自然に侵食され、廃墟になった文明の跡地でもなかった。
-It isn't the site of a fallen civilization that has been overtaken by nature, either.
+It isn't the site of a fallen civilization that has been overtaken by nature either.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_4.txt:187]: Pause feels more natural to me here

File: script/Ciel/Day 14/14_01B_03_CIEL14B_4_1D.txt [hunk 5]
```diff
@@ -261,7 +261,7 @@ The mighty claws of air arrive to crush my paltry resistance.
 [sha:59a807ea8d01fe14bb17a0b05c757a11e1d30ddb]{
 -- Page 85, Offset 40494.
 -- 「く―――そ！」
-"Ssshit!"
+"Shit!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_03_CIEL14B_4_1D.txt:264]: I think he _does_ kinda draw this one out

File: script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt [hunk 6]
```diff
@@ -329,7 +329,7 @@ I was holding back the whole time!
 [sha:c492d28ba3d85fe9b14be71848d34aefec36301d]{
 -- Page 305, Offset 41144.
 -- 「うっそつけぇ！」
-"Oh, bull%{i}shit%{/i}!"
+"Oh bull%{i}shit%{/i}!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt:332]: Prefer the pause personally

File: script/Ciel/Day 15/15_01A_CIEL15A_1.txt [hunk 7]
```diff
@@ -212,7 +212,7 @@ Clouds have begun to gather above us,
 -- Page 11, Offset 41418.
 -- 神父服の男がどんな<表|か><情|お>をしているのか、やはり読み取れない。
 // 神父服の feels a little clumsy here. Maybe we just shorted it to priest?
-I still really can't tell what kind of expression is on the priest's face.
+The priest's expression remains indistinguishable.
```
Comment (rschlaikjer) [inline script/Ciel/Day 15/15_01A_CIEL15A_1.txt:215]: Indistinguishable makes it sound like he can see it but not interpret it, but I think earlier he comments that he just can't get a clear look at his face at all?

> I still can't get a good look at the expression on the priest's face.

File: script/Ciel/Day 15/15_01A_CIEL15A_1.txt [hunk 8]
```diff
@@ -2684,7 +2684,7 @@ With that in mind, do you still plan on stepping into our world?
 [sha:64c755be6d60f606b040164582b1c1286d7c1312]{
 -- Page 156, Offset 41873.
 -- ―――捨てる必要のない、追われる必要もない、豊かな故郷がありながら」
-Despite having a home that you needn't either abandon or oust yourself from?"
+Despite having a home that you needn't abandon nor oust yourself from?"
```
Comment (rschlaikjer) [inline script/Ciel/Day 15/15_01A_CIEL15A_1.txt:2687]: Original sounds a slightly more normal to me

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 15/15_01A_CIEL15A_1.txt (hunk 7 above)
Comment (Petrikow) [reply script/Ciel/Day 15/15_01A_CIEL15A_1.txt:215]: It seems more like it's sort of blurry or indistinct
I think the original is fine

File: script/Ciel/Day 15/15_01A_CIEL15A_1.txt (hunk 8 above)
Comment (Petrikow) [reply script/Ciel/Day 15/15_01A_CIEL15A_1.txt:2687]: Hmmm, I could go with either, but the latter is shorter? idk

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt [hunk 9]
```diff
@@ -297,7 +297,7 @@ It is the voice of someone else coming from within me――from inside my own bo
 [sha:31b12a751e7d27b483788b3991ffccbd7478ff7d]{
 -- Page 48, Offset 39489.
 -- 「おまえ―――！」
-"You're―――!"
+"You bastard!"
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_02_CIEL14B_2B.txt:300]: no

**Pull request merged**